CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportController.java

```diff
@@ -488,6 +508,10 @@ public void setShowAllSchemaAttributes(final boolean showAllSchemaAttributes) {
         // TODO: the tick box could have changed but the menu item isn't updated, fix it
     }
 
+    public void setShowFilteredSchemaAttributes(final String filterAttributes) {
```

Comment:
This should be `setAttributeFilter`, and the variable should be `attributeFilter`.